pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "koerce"
-version = "0.0.1"
+version = "0.0.7"
```

Comment:
I tried following the poetry-dynamic-versioning guide: https://pypi.org/project/poetry-dynamic-versioning/

> Poetry still requires the tool.poetry.version field to be present in pyproject.toml, but you are encouraged to use version = "0.0.0" as a standard placeholder.

Could we use `0.0.0` here?

Reply:
Oh, yeah, probably.

I didn't really pay attention to whether dynamic verisoning was implemented and was just being lazy.

Reply:
Can you update to 0.0.0 then I would merge it.

Reply:
Done.